### PR TITLE
Allow using SELinux on image volume e2e test

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Enabling SELinux support for the e2e test without requiring a privileged container.
#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
PTAL @kubernetes/sig-node-pr-reviews

**Comment by saschagrunert** [comment]
PTAL @kubernetes/sig-node-pr-reviews

File: test/e2e_node/image_volume.go [hunk 1]
```diff
@@ -44,33 +44,49 @@ var _ = SIGDescribe("ImageVolume", nodefeature.ImageVolume, func() {
...
 		pod := &v1.Pod{
 			ObjectMeta: metav1.ObjectMeta{
 				Name:      podName,
 				Namespace: f.Namespace.Name,
 			},
 			Spec: v1.PodSpec{
 				RestartPolicy: v1.RestartPolicyAlways,
+				SecurityContext: &v1.PodSecurityContext{
+					SELinuxOptions: selinuxOptions,
+				},
```
Comment (kwilczynski) [inline test/e2e_node/image_volume.go:83]: Would it be OK if these options were set to some default values? Or would this also need a `if selinux.GetEnabled()` check?

Comment (saschagrunert) [reply test/e2e_node/image_volume.go:83]: I'd say we can assume that runtimes do not use the fields at all when SELinux is disabled, therefore we could keep it as is. Do you have thoughts on that?

Comment (kwilczynski) [reply test/e2e_node/image_volume.go:83]: No, that sounds good! Thank you!

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm
/approve

**Pull request merged**